loomi/regulariza-diagonal-client
# Feature/pending residents

## Feature
 - Pending Residents Page
  - Pending Residents Table
  - Pending Residents Virtual Table
  - Pending Residents Page Statistics
---

**What I did:**
 - Create the pending table, where the diagonal user can change the status of each answer and visualize all answers and file previews.
 - Create sidebar link to the pending residents
 


**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/32523401/130525384-eb1d881b-147e-4f16-b702-9ebbab2e3372.png)
